<launch>
    <node name="yolov5_node" type="rstest.py" pkg="human_following" output="screen"/>

    <node pkg="tf" type="static_transform_publisher" name="base_link_to_camera"
        args="0.12 0.04 0.76 0.0 -0.174 0.0 /base_link /camera_link 40" />

        <!-- args="0.12 0.04 0.76 0.0 -0.174 0.0 /base_link /camera_link 40" /> -->


</launch>